coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -1,3 +1,10 @@
+"""
+Documentation comments extraction, independently of language or docstyle.
+
+Each of the functions is built upon one another, and at the last,
+exposes a single public function `extract_documentation_with_markers`
```

Comment:
How can I see "compiled" documentation, taken from docstrings of panda? Should I run panda over itself? :)